src/ncp/ncp_base.cpp

```diff
@@ -1858,6 +1890,123 @@ ThreadError NcpBase::GetPropertyHandler_CNTR(uint8_t header, spinel_prop_key_t k
     return errorCode;
 }
 
+ThreadError NcpBase::GetPropertyHandler_MAC_WHITELIST(uint8_t header, spinel_prop_key_t key)
+{
+    otMacWhitelistEntry entry;
+    ThreadError errorCode = kThreadError_None;
+
+    SuccessOrExit(errorCode = OutboundFrameBegin());
+
+    SuccessOrExit(errorCode = OutboundFrameFeedPacked("Cii", header, SPINEL_CMD_PROP_VALUE_IS, key));
+
+    for (int i = 0; i < 32; i++)
+    {
```

Comment:
I know we have currently only 32 white list entries, so this is fine. 

Suggestion here to maybe change this to not have a `i < 32` condition in the for loop and let it breaks when we get an error code.  `otGetMacWhiteListEntry` should return `kThreadError_InvalidArgs` if we ask for a index higher than max.